src/main/java/seedu/address/logic/parser/AliasCommandParser.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.*;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.AliasCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AliasCommand object
+ */
+public class AliasCommandParser implements Parser<AliasCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AliasCommand
+     * and returns an AliasCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AliasCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_ALIAS_COMMAND,
```

Comment:
Could probably add in assertion or requireNonNull for defensive coding. Other than that, good job. Code looks neat and follow SLAP.